Makefile

```diff
@@ -2,9 +2,12 @@ CONTAINER_COMMAND = $(shell if [ -x "$(shell which docker)" ];then echo "docker"
 INSTALLER := $(or ${INSTALLER},quay.io/ocpmetal/assisted-installer:stable)
 GIT_REVISION := $(shell git rev-parse HEAD)
 CONTROLLER :=  $(or ${CONTROLLER}, quay.io/ocpmetal/assisted-installer-controller:stable)
+CONTROLLER_OCP :=  $(or ${CONTROLLER_OCP}, quay.io/ocpmetal/assisted-installer-controller-ocp:stable)
+CONTROLLER_OCP_IMAGE := $(or ${CONTROLLER_OCP_IMAGE},quay.io/ocpmetal/assisted-installer-controller-ocp:latest)
```

Comment:
why do we need both of them? why not just CONTROLLER_OCP

Reply:
copy-paste error :) will remove one.

Reply:
done

Reply:
Why do we need a new image?
Why not use the current controller image with a different entry point?
@gamli75 FYI :point_up:

Reply:
2 reasons:
1) we don't want to interfere with the day1 in anyway, so a new image will not affect anything, including DS buillds
2) currently we are delivering milestone1, which is mainly allowing to run on test-infra. the assisted-installer code won't go into production until we move both of them (service and installer) to be run as operators. This is a major task ( i am not even sure who will do it), and i am not even sure if it will use the same repo for it

Reply:
This change will require a new destination project in the midstream - @djzager can you review this PR? let's talk about the changes that needed for the DS.

Reply:
@gamli75 this new image needs not to be in DS, currently it will be used only in assisted-test-infra